### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
New lines added, new lines localized.

Check it please @jordimas @darccio @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -4964,7 +4964,7 @@ msgstr "Selecciona alguns d'aquests comptes per a seguir-los"
 
 #: src/screens/Onboarding/StepProfile/AvatarCreatorItems.tsx:83
 msgid "Select the {emojiName} emoji as your avatar"
-msgstr ""
+msgstr "Selecciona el {emojiName} emoji com al teu avatar "
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:4967]: The only tiny thing I spotted is that there's an extra space at the end here
```suggestion
msgstr "Selecciona el {emojiName} emoji com al teu avatar"
```

**Comment by jordimas** [comment]
My feedback is here: https://github.com/ivanbea/social-app/commit/f697a5d8b77ced7b51a3b41d969b4e2fd70de565

**Comment by jordimas** [comment]
It will be good to address the issues here:

https://static.softcatala.org/quality/social-app.html

For example, most of these are spelling mistakes:

_Tot minúscules: bsky, captcha, categoritxar, dessitjat, enganyòs, etiquetador, etiquetadors, interactui, interesos, jordi,

Primera lletra majúscula: Aqeusta, Communta, Etiquetador, Introdeix,_

**Comment by ivanbea** [comment]
Fet, moltíssimes gràcies @jordimas

**Review by pfrazee — APPROVED** [review]
Thanks all!

**Pull request merged**